src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    if (robot[key] in invertedRobot) {
```

Comment:
The check `robot[key] in invertedRobot` is using the `in` operator, which checks for the presence of a key in an object. However, since `robot[key]` is a value and not a key, you should use `hasOwnProperty` to check if the value already exists as a key in `invertedRobot`.